.github/workflows/python-app.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: offer-fit-integration
+
+on: push
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Have you had a look at the Github Actions [documentation on running CI from a Docker image](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action#creating-an-action-metadata-file)?

Reply:
Why define and push a separate action for this? Isn't it simpler to have everything in the workflow for a project like this? In this case my python-app-docker.yml is the one running a container.